.github/workflows/main.yml

```diff
@@ -20,15 +20,15 @@ jobs:
     name: Test Suite for encoded-core
 
     # The type of runner that the job will run on
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.9
```

Comment:
I'd like to see this tested in the low and high version allowed, so 3.8 and 3.11.

Reply:
Done